newdle/api.py

```diff
@@ -291,6 +292,21 @@ def get_newdles_participating():
     return NewdleParticipantSchema(many=True).jsonify(newdle)
 
 
+@api.route('/newdle/event', methods=('POST',))
+@use_kwargs(NewEventSchema(), locations=('json',))
+def create_event(title, duration, final_dt):
+    url = current_app.config['INDICO_URL']
+    res = requests.post(
+        url,
+        data={
+            'title': title,
+            'start_dt': final_dt,
+            'duration': duration,
+        },
+    )
+    return res.text
```

Comment:
Add `res.raise_for_status()` to fail if the request is not successful.

Reply:
Thanks, added.